pom.xml

```diff
@@ -145,6 +146,8 @@
     <jgit.version>5.12.0.202106070339-r</jgit.version>
     <jjwt.version>0.11.2</jjwt.version>
     <jsr305.version>3.0.2</jsr305.version>
+    <jgit.version>5.12.0.202106070339-r</jgit.version>
+    <jsr305.version>3.0.2</jsr305.version>
```

Comment:
looks like some leftovers from rebasing for `jgit` & `jsr305`

Reply:
fixed